spoc/models/dataframe_models.py

```diff
@@ -72,6 +72,10 @@ def get_region_number(self) -> Optional[int]:
         """Returns the number of regions in the genomic data
         if present."""
 
+    def get_window_size(self) -> Optional[int]:
```

Comment:
Yeah, it's true. At the moment `GenomicDataSchema` carries all information about the accumulating data. In the end, we need such an object to perform validation. I don't really have an idea about how to do it otherwise. I think it's fair in principle since in a sense the genomicdata that we accumulate are tied to a specific windowsize. It has been imparted by overlapping the region, but the result is linked to it.